flipperdevices/flipperzero-firmware
# [FL-1587] RFID: Clock for emulation timer from antenna

# What's new

- Add F7 target
- New RFID emulation scheme

# Verification 

- Compile and upload
- Check F7 (DVT batch)
- Try emulating different RFID cards with different readers

# Checklist (do not modify)
